README.md

```diff
@@ -61,6 +65,51 @@ You can also always find the full command-line options that are available with
 $ cargo fuzz --help
 ```
 
+## Code coverage
+
+### Prerequisites
+Install the LLVM-coverage tools as described in the [Unstable book](https://doc.rust-lang.org/beta/unstable-book/compiler-flags/source-based-code-coverage.html#installing-llvm-coverage-tools).
+
+We recommend using at least LLVM 11 and a recent nightly version of the Rust toolchain.
+This code was tested with `1.51.0-nightly (2021-02-10)`.
+
+### Generate code-coverage data
+
+After you fuzzed your program, use the `coverage` command to generate precise
+[source-based code coverage](https://blog.rust-lang.org/inside-rust/2020/11/12/source-based-code-coverage.html)
+information:
+```
+$ cargo fuzz coverage <target> [corpus dir]
+```
+This command
+
+- compiles your project using the `-Zinstrument-coverage` Rust compiler flag,
```

Comment:
Are there any issues with running `fuzz coverage` whilst fuzzing is ongoing (e.g. in another terminal)?
Similarly with alternating between fuzz run and fuzz coverage?

If it potentially causes issues it would be helpful to document them (if we can't work around them with, say, separate target binaries for coverage-instrumented vs fuzzing-instrumented).

Reply:
I tried running `cargo fuzz run` and `cargo fuzz coverage` on the same target (a WebAssembly interpreter) in parallel and didn't encounter issues. The `coverage` command generated information for the corpus data that was available at the moment it was run, and I was able to re-run it while the fuzzer was still running. I was also able to re-run the fuzzer while the coverage information was being produced. 
However, note that alternating between the `run` and `fuzz` commands involves rebuilding the target because they require different compiler options.
Are there more potential issues you would be looking for? I imagine that in general one would want to avoid running the two modes in parallel to maximize the computing resources available to the fuzzer.

Reply:
> I was also able to re-run the fuzzer while the coverage information was being produced.

I don't fully understand how this works, but if it does then that's great!

My confusion is that there is only a single target binary path, so it seems we either have a binary with coverage instrumentation or not, and will never have both at the same time. In the scenario you tested, how do we know which version is going to get run? Might the coverage loop switch to the non-coverage-instrumented binary half way through, when it gets compiled for `cargo fuzz run`?

Reply:
That's a good point. I decided to just in case create a separate build target for `cargo fuzz coverage`.